html/semantics/embedded-content/the-img-element/relevant-mutations.html

```diff
@@ -159,43 +159,43 @@
 
   t('crossorigin absent to empty', function(img) {
     img.crossOrigin = '';
-  }, 'error');
+  }, 'timeout');
 
   t('crossorigin absent to anonymous', function(img) {
     img.crossOrigin = 'anonymous';
-  }, 'error');
+  }, 'timeout');
 
   t('crossorigin absent to use-credentials', function(img) {
     img.crossOrigin = 'use-credentials';
-  }, 'error');
+  }, 'timeout');
 
   t('crossorigin empty to absent', function(img) {
     img.removeAttribute('crossorigin');
-  }, 'error');
+  }, 'timeout');
 
   t('crossorigin empty to use-credentials', function(img) {
     img.crossOrigin = 'use-credentials';
-  }, 'error');
+  }, 'timeout');
```

Comment:
I'm not sure I understand this test. Are these tests run serially? Is `img` always the same image.
If so, I think I'd expect 'use-credentials' here to trigger a "relevant mutation" which would trigger a load (with different cross origin headers) and would eventually trigger a load event.